src/main/java/seedu/address/model/person/Person.java

```diff
@@ -93,14 +87,13 @@ public boolean equals(Object other) {
         return name.equals(otherPerson.name)
                 && phone.equals(otherPerson.phone)
                 && email.equals(otherPerson.email)
-                && address.equals(otherPerson.address)
                 && tags.equals(otherPerson.tags);
     }
 
     @Override
     public int hashCode() {
         // use this method for custom fields hashing instead of implementing your own
-        return Objects.hash(name, phone, email, address, tags);
+        return Objects.hash(name, phone, email, tags);
```

Comment:
for the codecov warning try to add/modify a test case in `/test/java/seedu/address/model/person/PersonTest.java` (?) very weird since there wasn't even a test case when i checked the file

Reply:
Okay I did this but now it fails for another reason LOL and this time idk why it fails..

Reply:
https://github.com/nus-cs2103-AY2324S1/forum/issues/248

apparently prof said that drop in coverage is sometimes inevitable so merging is fine